pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -307,6 +309,52 @@ func (l *LibvirtDomainManager) setGuestTime(vmi *v1.VirtualMachineInstance) erro
 	return nil
 }
 
+func (l *LibvirtDomainManager) reconnectGuestNics(vmi *v1.VirtualMachineInstance) error {
+	// Set interfaces link down and up to renew DHCP leases
+
+	domName := api.VMINamespaceKeyFunc(vmi)
+	dom, err := l.virConn.LookupDomainByName(domName)
+	if err != nil {
+		log.Log.Object(vmi).Reason(err).Error(failedReconnectGuestNic)
+		return err
+	}
+	xmlstr, err := dom.GetXMLDesc(0)
+	if err != nil {
+		return err
+	}
+
+	var domain api.DomainSpec
+	err = xml.Unmarshal([]byte(xmlstr), &domain)
+	if err != nil {
+		return fmt.Errorf("parsing domain XML failed, err: %v", err)
+	}
+
+	//Look up all the interfaces and reconnect them
+	for _, iface := range domain.Devices.Interfaces {
```

Comment:
shouldn't we skip the pod network ?

Reply:
It was mainly done for pod network, but some other cni can also change the pod IP/Mac address after the migration